client/src/app.html

```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <meta name="viewport" content="width=device-width" />
+
+    <title>Cat Me if You Can</title>
```

Comment:
`if` should probably be with a capital I

Reply:
fixed in cbb652a853949b25251a5c39c26d1ee8c1a973dc